### PR TITLE
💄(UI) Registraion and login error fixes and updates

### PR DESCRIPTION
**Pull request by ricecakesreece**

### CODE REVIEW

File: libs/shared/components/location-field/src/lib/location-field.component.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+    for (let j = 0; j < len; j++) 
+    { 
+      if(this.potentialLocations[j] == this.val){
+        return true;
+      }
+    }
+    return false;
+  }
+}
```
Comment (Ruben-Brits) [inline libs/shared/components/location-field/src/lib/location-field.component.ts:120]: empty line end of file needed

**Review by Ruben-Brits — CHANGES_REQUESTED** [review]
forgot password.spec file has unused variables as well, couldn't comment on them for some reason. Please remove them

**Review by Ruben-Brits — APPROVED** [review]

File: libs/login/feature/src/lib/login.component.scss [hunk 2]
```diff
@@ -121,14 +129,4 @@ ion-content {
...
-.error{
-    display: none;
-    position: relative;
-    display: block;
-    text-align: center;
-    margin: 0;
-    top: 27px;
-    color: red;
-}
+}
```
Comment (PxttytheThxtty) [inline libs/login/feature/src/lib/login.component.scss:132]: Newline EOF

File: libs/login/feature/src/lib/login.component.ts [hunk 3]
```diff
@@ -155,18 +128,20 @@ export class LoginComponent implements OnInit {
         error => {
           console.log("Error has occured with API: " + error);
         }
-      )
+      );
 
       if(id == "")
       {
-        this.openToast("Inccorect email or password");
+        this.errStatement = "Inccorect email or password";
```
Comment (PxttytheThxtty) [inline libs/login/feature/src/lib/login.component.ts:135]: Incorrect is spelt wrong

File: libs/shared/components/input-field/src/lib/input-field.component.scss [hunk 4]
```diff
@@ -43,16 +43,32 @@
...
+    padding: 0;
+    height: 0;
+    margin-top: -0.5rem;
+}
+
+.errorType p {
+    color: red;
+    text-align: center;
+    font-size: 15px;
 }
```
Comment (PxttytheThxtty) [inline libs/shared/components/input-field/src/lib/input-field.component.scss:74]: Newline at EOF

File: libs/shared/components/password-field/src/lib/password-field.component.html [hunk 5]
```diff
@@ -6,4 +6,6 @@
         <ion-icon class="show-option" [hidden]="showPassword" name="eye-off-outline"></ion-icon>
         <ion-icon class="hide-option" [hidden]="!showPassword" name="eye-outline"></ion-icon>
     </a>
+
+    <div class="errorType"><p [hidden]="!showError">{{ errorContent }}</p></div> 
 </div>
```
Comment (PxttytheThxtty) [inline libs/shared/components/password-field/src/lib/password-field.component.html:11]: No newline at EOF

File: libs/shared/components/password-field/src/lib/password-field.component.scss [hunk 6]
```diff
@@ -44,20 +44,35 @@
...
+    margin: 0;
+    padding: 0;
+    height: 0;
+    margin-top: -0.5rem;
+}
+
+.errorType p {
+    color: red;
+    text-align: center;
 }
```
Comment (PxttytheThxtty) [inline libs/shared/components/password-field/src/lib/password-field.component.scss:78]: No newline at EOF

**Review by PxttytheThxtty — CHANGES_REQUESTED** [review]

**Review by Ruben-Brits — DISMISSED** [review]

**Comment by priyo5** [comment]
Au pairs need to have fields for bio and experience so the admin can view why they want to be an au pair and their qualifications

**Review by priyo5 — APPROVED** [review]

**Review by Ruben-Brits — APPROVED** [review]

**Review by PxttytheThxtty — APPROVED** [review]

**Pull request merged**